travis_pypi_setup.py

```diff
@@ -94,29 +95,34 @@ def update_travis_deploy_password(encrypted_password):
     """
     config = load_yaml_config(TRAVIS_CONFIG_FILE)
 
-    config['deploy']['password'] = dict(secure=encrypted_password)
+    config["deploy"]["password"] = dict(secure=encrypted_password)
 
     save_yaml_config(TRAVIS_CONFIG_FILE, config)
 
-    line = ('# This file was autogenerated and will overwrite'
-            ' each time you run travis_pypi_setup.py\n')
+    line = (
+        "# This file was autogenerated and will overwrite"
+        " each time you run travis_pypi_setup.py\n"
+    )
     prepend_line(TRAVIS_CONFIG_FILE, line)
 
 
 def main(args):
     public_key = fetch_public_key(args.repo)
-    password = args.password or getpass('PyPI password: ')
+    password = args.password or getpass("PyPI password: ")
     update_travis_deploy_password(encrypt(public_key, password.encode()))
     print("Wrote encrypted password to .travis.yml -- you're ready to deploy")
 
 
-if '__main__' == __name__:
+if "__main__" == __name__:
```

Comment:
[inline-plz]: `prospector: pylint: Comparison should be __name__ == '__main__' (misplaced-comparison-constant)`